### ISSUE TITLE
cannot use CloneApp with screen readers 

### ISSUE THREAD

**Issue by MesterPerfect** [mirinsoft/CloneApp#21]
Hello
I cannot use CloneApp with screen readers
I can navigate all the buttons in the program but the screen reader only reads the list of programs and the location of the search
